lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.ts

```diff
@@ -119,7 +120,7 @@ export class SearchFilterContainerComponent implements OnInit, OnDestroy {
     }
 
     isActive(): boolean {
-        return this.widgetContainer && this.widgetContainer.componentRef && this.widgetContainer.componentRef.instance.isActive;
+        return this.searchFilterQueryBuilder.getActiveFilters().findIndex((f: FilterSearch) => f.key === this.category.columnKey) > -1;
```

Comment:
given this is declared a bug fix, it needs corresponding unit tests as a proof :)

Reply:
I will try my best to think of a test I can do for this then